.github/workflows/triage_add_to_routed_project.yml

```diff
@@ -1,5 +1,10 @@
 name: 'Triage: route to team project board'
 on:
+# makes this workflow reusable
+  workflow_call:
+    secrets:
+      ADD_TO_TRIAGE_BOARD_TOKEN:
```

Comment:
Not necessary on this PR but we should probably change the name of this token since it's used for more than just adding to the triage project.

Reply:
yeah I was thinking about that.  The other uses are less "invasive" in my opinion like moving a ticket between columns on the project board so I think it is ok for the moment that the ticket at least shows its most powerful permission in the name.